Java/libraries/recognizers-text-datatypes-timex-expression/src/main/java/com/microsoft/recognizers/datatypes/timex/expression/TimexFormat.java

```diff
@@ -157,17 +158,24 @@ private static String formatDateRange(TimexProperty timex) {
         }
 
         if (timex.getMonth() != null && timex.getWeekOfMonth() != null && timex.getDayOfWeek() != null) {
-            return String.format("XXXX-%1$s-WXX-%2$s-%3$s", TimexDateHelpers.fixedFormatNumber(timex.getMonth(), 2),
+            return String.format("%1$s-%2$s-%3$s-%4$s",
```

Comment:
I don't know very well how the string format works, but here, isn't missing a specifier inside the string since you have five variables?

Reply:
You are right @fran893 , this should be changes to `%1$s-%2$s-%3$s-%4$s-%5$s`

Reply:
my bad, I miscounted four since I left 2 in the same line